packages/analytics-browser/test/helpers/mock.ts

```diff
@@ -54,6 +54,7 @@ export const createConfigurationMock = (options?: Partial<BrowserConfig>) => {
       send: jest.fn(),
     },
     useBatch: false,
+    diagnosticProvider: new Diagnostic(),
```

Comment:
Should this be a mock object like the other providers?

Reply:
Changed in https://github.com/amplitude/Amplitude-TypeScript/pull/568/commits/860bde26d816642ae744ad8d4d14c214f805d6ae